database/schema.sql

```diff
@@ -0,0 +1,7 @@
+DROP DATABASE IF EXISTS  user_reviews;
+CREATE DATABASE user_reviews;
+USE user_reviews;
+
+/*  Execute this file from the command line by typing:
```

Comment:
I really like your detailed comments. Really helps reviewers understand your thought process and your code.